sdk/nodejs/serverless/function.ts

```diff
@@ -254,55 +270,51 @@ interface Package {
 
 // allFolders computes the set of package folders that are transitively required by the given list of package
 // dependencies rooted in a package at the provided path.
```

Comment:
This comment looks like it should be update to reflect the new behaviour.

Reply:
done.